Makefile.am

```diff
@@ -200,3 +212,16 @@ distcheck-hook:
 
 tidyall:
 	tidyall --conf-file doc/dot.tidyallrc --root-dir . -r src t xt
+
+all-local: sympa.conf-dist
+
+sympa.conf-dist: mk-sympa-dist.pl
+	@./mk-sympa-dist.pl
+
+mk-sympa-dist.pl:
+	@rm -f $@
+	$(AM_V_GEN)$(SED) \
+		-e 's|--PERL--|$(PERL)|' \
+		-e 's|--srcdir--|$(srcdir)|' \
+		< $(srcdir)/$@.in > $@
+	@chmod +x $@
```

Comment:
- mk-sympa-dist.pl may be invoked doing like “$(PERL) mk-sympa-dist.pl” (executable bit is not always set).
- It can also be generated by adding *.in to configure.ac (e.g. sympa.conf is generated in that way). Although I can’t decide which way is better.

Reply:
Well, your way to do makes less code, so let's do it your way.
Fixed.

Reply:
Good.